website/content/docs/getting-started.md

```diff
@@ -175,6 +196,59 @@ The included example topologies:
 * `TaskHookTopology.java` --- A topology that uses a task hook to subscribe to
    event notifications.
 
+## Frequently Asked Questions
+
+### 1. How do I know if my topology has been successfully submitted?
+
+- The following message would appear upon successful submission:
+
+```bash
+  INFO: Topology 'ExclamationTopology' launched successfully
+```
+- Note even if the topology is submitted successfully, it could still fail to start some component. For example, `stmgr` may fail to start due to unfulfilled dependencies. To troubleshoot, refer to question 3 and the debugging example below.
```

Comment:
Link to 'question 3'.

Also pls limit line lengths to 100 to make it easier to comment and read in an editor.